src/api/glue-api.js

```diff
@@ -0,0 +1,10 @@
+import { get } from '../helpers/request-helper'
+
+const ROOT_URL = 'https://glue-api.zapimoveis.com.br/v2/listings?categoryPage=RESULT'
+const villageHouse = '&unitSubTypes=VILLAGE_HOUSE&unitTypes=HOME&usageTypes=RESIDENTIAL&unitTypesV3=VILLAGE_HOUSE'
+
+export async function getListings(searchTerm) {
+  const searchTermStringified = new URLSearchParams(searchTerm)
+  const response = await get(`${ROOT_URL}${villageHouse}${searchTermStringified}`)
```

Comment:
You can create try and catch to treated this await. 
In case the API send error, you can get. What do you think ?

Reply:
Like this?
`export function getListings(searchTerm) {
  const searchTermStringified = new URLSearchParams(searchTerm)

  try {
    const response = get(`${BASE_URL}${villageHouse}${searchTermStringified}`)
    return response.data.search.result.listings
  } catch (error) {
    return error
  }
}`

Reply:
What do you think to returns an empty Array when the API is fail?

```javascript
try {
    const response = await get(${BASE_URL}${villageHouse}${searchTermStringified})
    return response.data.search.result.listings
} catch (error) {
    console.error(error)
    return []
}
```

Reply:
I like it.